src/layouts/components/Editor/extensions/Details/IsomerDetailGroup.ts

```diff
@@ -32,7 +33,7 @@ export const IsomerDetailsGroup = Node.create<IsomerDetailGroupOptions>({
   atom: true,
   draggable: true,
   content: "details+",
-  defining: true,
+  definingForContext: true,
```

Comment:
this is new to me, could you eli5 this for me? couldn't seem to find it on the tiptap docs oddly

Reply:
sure, this is a prosemirror thing found [here](https://prosemirror.net/docs/ref/#model.NodeSpec.definingForContent). TLDR, had this weird bug when if you backspace on an accordion title the format screws up, so this flag j sets the parent as immutable